src/include/rbd/librbd.h

```diff
@@ -189,7 +189,8 @@ typedef enum {
 typedef enum {
   RBD_MIRROR_IMAGE_DISABLING = 0,
   RBD_MIRROR_IMAGE_ENABLED = 1,
-  RBD_MIRROR_IMAGE_DISABLED = 2
+  RBD_MIRROR_IMAGE_DISABLED = 2,
+  RBD_MIRROR_IMAGE_CREATING = 3
```

Comment:
Let's expose it in Python bindings too (`src/pybind/rbd/c_rbd.pxd`, `src/pybind/rbd/mock_rbd.pxi` and `src/pybind/rbd/rbd.pyx`).